fastlane/lib/fastlane/actions/actions_helper.rb

```diff
@@ -134,5 +134,14 @@ def self.load_external_actions(path)
     def self.formerly_bundled_actions
       ["xcake"]
     end
+
+    def self.is_class_action?(class_ref)
+      return false if class_ref.nil?
+      class_ref < Fastlane::Action || false
```

Comment:
That's a little confusing, can you use a regular if else block to make it easier to understand for contributors?